### DISCUSSION TITLE
Welcome to APEX_yolov5_assist Discussions!

### DISCUSSION THREAD

**Discussion by brillian32** [brillian32/APEX_yolov5_assist#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xac19283tsai** [comment]
你好我想要跟你拿權重檔或訓練資料

**Comment by zhaoyihe031** [comment]
您好，我是何炤毅，你的来件我已收到，我会在阅读后尽快给您回复。谢谢~
